src/MusicDrawer/styled-components.js

```diff
@@ -72,7 +71,6 @@ export const NowPlayingDrawer = styled.div`
 
 export const ClosedControlsContainer = styled.div`
   position: absolute;
-  right: 0;
```

Comment:
`right` style duplicate with **line 77**, and I think `12px` is more appropriate.